controlplane/hack/deploy/addon/policy/hub/kustomization.yaml

```diff
@@ -0,0 +1,3 @@
+resources:
+# - clustermanagementaddon
+- crds
```

Comment:
should add clustermanagementaddon CR in hub

Reply:
ignore.  I will add it in the addon controler pr.

Reply:
it should be managed by policy addon.